src/ray/protobuf/gcs_service.proto

```diff
@@ -587,7 +593,8 @@ message GetAllResourceUsageRequest {
 
 message GetAllResourceUsageReply {
   GcsStatus status = 1;
-  ResourceUsageBatchData resource_usage_data = 2;
+  bool cluster_full_of_actors_detected_by_gcs = 2;
```

Comment:
Could you append this as the third message rather than inserting it as the second?

Reply:
Same applies to all fields added in this PR.

Reply:
Sure, will do.